command/alloc_exec.go

```diff
@@ -230,7 +234,7 @@ func (l *AllocExecCommand) Run(args []string) int {
 
 // execImpl invokes the Alloc Exec api call, it also prepares and restores terminal states as necessary.
 func (l *AllocExecCommand) execImpl(client *api.Client, alloc *api.Allocation, task string, tty bool,
-	command []string, escapeChar string, stdin io.Reader, stdout, stderr io.WriteCloser) (int, error) {
+	command, envs []string, escapeChar string, stdin io.Reader, stdout, stderr io.WriteCloser) (int, error) {
```

Comment:
```suggestion
	command []string, envs []string, escapeChar string, stdin io.Reader, stdout, stderr io.WriteCloser) (int, error) {
```

Reply:
Go allows skipping redundant type declaration when multiple func parameters have the same type, so the code is ok as-is.